src/WinUIEx/HwndExtensions.cs

```diff
@@ -115,6 +115,161 @@ public static void CenterOnScreen(IntPtr hwnd, double? width = null, double? hei
             SetWindowPosOrThrow(new HWND(hwnd), new HWND(), left, top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
         }
 
+        /// <summary>
+        /// Moves the window to the specified coordinates
+        /// </summary>
+        /// <param name="hwnd">Window handle</param>
+        /// <param name="x">Left side of the window</param>
+        /// <param name="y">Top side of the window</param>
+        public static void MoveToPoint(IntPtr hwnd, double x, double y)
+        {
+            PInvoke.GetWindowRect(new HWND(hwnd), out RECT windowRect);
+            var w = windowRect.right - windowRect.left;
+            var h = windowRect.bottom - windowRect.top;
+            var left = x;
+            var top = y;
+            SetWindowPosOrThrow(new HWND(hwnd), new HWND(), (int)left, (int)top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="hwnd"></param>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="width"></param>
+        /// <param name="height"></param>
+        public static void CenterOnPoint(IntPtr hwnd, double x, double y, double? width = null, double? height = null)
+        {
+            var dpi = PInvoke.GetDpiForWindow(new HWND(hwnd));
+            PInvoke.GetWindowRect(new HWND(hwnd), out RECT windowRect);
+            var scalingFactor = dpi / 96d;
+            var w = width.HasValue ? (int)(width * scalingFactor) : windowRect.right - windowRect.left;
+            var h = height.HasValue ? (int)(height * scalingFactor) : windowRect.bottom - windowRect.top;
+            var cx = x;
+            var cy = y;
+            var left = cx - (w / 2);
+            var top = cy - (h / 2);
+            SetWindowPosOrThrow(new HWND(hwnd), new HWND(), (int)left, (int)top, w, h, SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW);
+        }
+
+
+
+
+        /// <summary>
+        /// Moves the window to the specified enum edge position on the current monitor.
+        /// </summary>
+        /// <param name="hwnd">The window handle</param>
+        /// <param name="position">The enum edge position from Positions</param>
+        /// <param name="forceTowardsCenter">Forces the window away from edge and towards the center by absolute XY values</param>
```

Comment:
I don't really understand this parameter. Isn't that what the offsets already do?

Reply:
When forceToCenter is true, the offsets will effectively be an absolute margin that remains no matter where it is moved on screen (or if the screen was rotated).  It has the added bonus of making sure the window isn't moved somewhere out of view.

For example, if I wanted to move my window from the MiddleCenter to the left all I'd need to do is change one single Placement Parameter and it'd move but remain the exact same distance from the edge of the screen.